sos/collector/clusters/__init__.py

```diff
@@ -81,6 +82,107 @@ def name(cls):
+            section.add_text(cls.__mro__[1].__doc__)
+        else:
+            section.add_text(
+                "\n\tDetailed help not available for this profile\n"
+            )
+
+        if cls.packages:
+            section.add_text(
+                "Enabled by the following packages: %s"
+                % ', '.join(p for p in cls.packages),
+                newline=False
+            )
+
+        if cls.sos_preset:
+            section.add_text(
+                "Uses the following sos preset: %s" % cls.sos_preset,
+                newline=False
+            )
+
+        if cls.sos_plugins:
+            section.add_text(
+                "Enables the following plugins: %s"
+                % ', '.join(plug for plug in cls.sos_plugins),
+                newline=False
+            )
+
+        if cls.sos_plugin_options:
+            _opts = cls.sos_plugin_options
+            opts = ', '.join("%s=%s" % (opt, _opts[opt]) for opt in _opts)
+            section.add_text(
+                "Sets the following plugin options: %s" % opts,
+                newline=False
+            )
+
+        if cls.option_list:
+            optsec = section.add_section("Available cluster options")
+            optsec.add_text(
+                "These options may be toggled or changed using '%s'"
+                % bold("-c %s.$option=$value" % cls.__name__)
+            )
+            optsec.add_text(bold(
+                "\n{:<4}{:<20}{:<30}{:<20}\n".format(
+                    ' ', "Option Name", "Default", "Description")
+                ), newline=False
+            )
+            for opt in cls.option_list:
+                val = opt[1]
+                if isinstance(val, bool):
+                    if val:
+                        val = 'True/On'
+                    else:
+                        val = 'False/Off'
+                _ln = "{:<4}{:<20}{:<30}{:<20}".format(' ', opt[0], val,
+                                                       opt[2])
+                optsec.add_text(_ln, newline=False)
+
+    @classmethod
+    def display_self_help(cls, section):
+        section.set_title('SoS Collect Cluster Profiles Detailed Help')
+        section.add_text(
+            '\nCluster profiles are used to represent different clustering '
+            'technologies or platforms. Profiles define how cluster nodes are '
+            'discovered, and optionally filtered, for default executions of '
+            'collector.'
+        )
+        section.add_text(
+            'Cluster profiles are enabled similarly to SoS report plugins; '
+            'usually by package, command, or configuration file presence. '
+            'Clusters may also define default transports for SoS collect.'
+        )
+
+        from sos.collector import SoSCollector
+        import sos.collector.clusters
+        clusters = SoSCollector._load_modules(sos.collector.clusters,
+                                              'clusters')
```

Comment:
LGTM complains here on self-import. Not sure how legitimate it is, but an alternative here is:

        import inspect
        clusters = SoSCollector._load_modules(inspect.getmodule(cls),
                                              'clusters')

Reply:
:+1: